scripts/common.py

```diff
@@ -50,7 +50,7 @@
     OPPIA_TOOLS_DIR, 'google-cloud-sdk-251.0.0', 'google-cloud-sdk')
 NODE_PATH = os.path.join(OPPIA_TOOLS_DIR, 'node-%s' % NODE_VERSION)
 NODE_MODULES_PATH = os.path.join(CURR_DIR, 'node_modules')
-FRONTEND_DIR = os.path.join(CURR_DIR, 'core', 'templates', 'dev', 'head')
+FRONTEND_DIR = os.path.join(CURR_DIR, 'core', 'templates', '')
```

Comment:
Why is there an empty string at the end?

Reply:
fixed